tests/flow/test_bfs.py

```
@@ -48,46 +32,104 @@
+            """MATCH (a {v: 'a'}) CALL algo.bfs(null) YIELD nodes, invalidField""",
+
+            # max depth is expected to an integer
+            # FIXME should throw error but does not:
+            # """MATCH (a {v: 'a'}) CALL algo.bfs(a, 'hello', NULL)""",
+        ]
+
+        for q in invalid_queries:
+            try:
+                self.graph.query(q)
+                self.env.assertFalse(True)
+            except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)